_docs/whatiscloud.md

```diff
@@ -0,0 +1,45 @@
+
+### Intriguing...so what can I do in the cloud?
+Almost anything you can dream of. Cloud providers are constantly improving and adding to their offerings. A good place to start would be to take a simple application that you could run locally, such as an app that retrieves and stores information in a database and move it into the cloud. Refer to [LINK HERE] for a few concrete examples.
+
+### Wait, there's more than one cloud provider?
+You bet. Amazon Web Services (AWS), Microsoft Azure, and Google Cloud Platform (GCP) are the three most well-known cloud providers out there today, but they aren’t the only ones. In order to keep these docs coherent (and us, sane) we’ll be focusing on these three.
+
+
+### Why should I care about the cloud? It works fine on my machine!
+Try shipping your machine to the customer, then get back to me. Really though, it’s because the applications that you can create in the cloud can be far more capable than what you can create on your machine. Web hosting, machine learning, databases, serverless functions and more are at your fingertips allowing you to build applications in a way you never could before.
+
+### Alright, you've convinced me. So how do I get started?
+Head to [LINK HERE] and follow the exquisitely detailed instructions to learn how to signup for a free account on each cloud platform. It is highly recommended to peruse the documentation for each offering you want to use. Links to AWS, Azure, and GCP documentation websites have been provided for your convenience.
+* AWS Documentation [LINK]
+* Azure Documentation [LINK]
+* GCP Documentation [LINK]
+
+### Is this stuff really free???
+There is no such thing as a free lunch. More to the point though, each provider has a set of “always free” offerings, and a set of free trial offerings. All free offerings have usage limitations which will be clearly marked on the provider’s website. **Exceeding the listed limitations will result in charges which you will be responsible for.** Cloud providers will email you when you use credits or have a new invoice posted. Keep track of these communications and avoid leaving services running in order to minimize potential charges. Free trials typically last 12 months, after which you will be charged for any offering that is not in the “always free” category. Student accounts can often be renewed at the provider’s discretion.
+
+### I heard there were free credits. How do I get some?
+Refer to [INSERT LINK HERE] to learn how you can get your hands on some free student credits!
```

Comment:
```suggestion
Refer to [our guide on cloud providers](_docs/cloud_account_setup.md) to learn how you can get your hands on some free student credits!
```

Reply:
Note: I'm not sure how the internal linking will work just yet; it might have to be linked to the `.html` permalink address instead of the `.md` file